docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,205 @@
+[[synthetic-run-tests]]
+= Run synthetic tests
+
+++++
+<titleabbrev>Run synthetic tests</titleabbrev>
+++++
+
+There are two ways to run synthetic tests:
+
+* If you want to create a single journey and manage it in isolation from monitors containing other journeys,
+<<synthetics-inline-journey,use an inline journey>>.
+Inline journeys can be more efficient, but have some limitations like not supporting dependencies or multiple journeys.
+* If you have multiple journeys, rely on dependencies, or your tests need to live with your
+application code, <<synthetics-test-suite,use a test suite>>.
```

Comment:
I'd probably be a bit more precise with what we mean by efficient, otherwise it's kind of an empty statement IMO:

```suggestion
Inline monitors are quick and easy to setup. To create one, you can copy a journey's content, paste it into Kibana, and hit save. However, this type of monitor does not allow you to import third-party packages.

Inline monitors can also be more difficult to manage because each inline monitor can contain only _one_ journey, which must be maintained from within Kibana. Therefore, if you depend on external packages, have your journeys next to your code repo, or just wish to embed and manage multiple journeys from a single monitor configuration <<synthetics-test-suite,use a test suite>>.
```

PS: Here I'm also using the term "inline _monitor_". A journey is the code, while the monitor is what lives in Kibana.